base/service.sh

```diff
@@ -17,4 +17,7 @@ done
 sleep 5
 
 # restart on crash
-frida-server -D
+while true; do
```

Comment:
@enovella I remember you had issues with this loop. Can you please test if this is still the case with this PR?

Reply:
Magiskfrida will become meaningless if it is not looped, and it can work normally. In fact, the listening port should be modified here, but it is no problem to use -D for the public.

Reply:
Two potential issues here:
1. `-D` will run Frida in daemon mode, which means it will fork the process and continue the script immediately. So, every 2 seconds, a new Frida process will spawn. We already had this accidentally happen, and I had to revert it: d0eee42cb42332995c5b20f6a8e7ab3c8d5afb76 / https://github.com/ViRb3/magisk-frida/pull/18#issuecomment-1022078587
2. If you remove `-D`, then this becomes the loop implementation I had not too long ago. However, @enovella reported that with this loop, Frida wouldn't work on any of their devices. I really have no idea why that would be the case, but we need to sort this out before we introduce it back in. Ref: https://github.com/ViRb3/magisk-frida/pull/46#issuecomment-1944525071

Reply:
It is more appropriate to change it to 5 seconds. In addition, it is ideal to change the script to detect that the process does not exist and then pull it up. If there is no loop, then this module is meaningless.

Reply:
Even if we change it to 5 seconds, if we keep the `-D` flag, a new process will start regardless of whether the previous crashed or not. I also believe the loop should be there, just without the `-D` flag, and then we don't need to bother with somehow tracking if the process died or not. I just want to make sure that this works for @enovella's devices.

Reply:
Well, it's ok to get stuck there.

Reply:
-l 0.0.0.0:27042 In this way, it will not cause the system to crash.